operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -634,15 +634,17 @@ func (c *cesMgr) insertCESInWorkQueue(ces *cesTracker, baseDelay time.Duration)
 
 // Return the CES queue delay in seconds and reset cesInsert time.
 func (c *cesMgr) getCESQueueDelayInSeconds(cesName string) (diff float64) {
-	ces := c.desiredCESs.getCESTrackerOnly(cesName)
-	ces.backendMutex.Lock()
-	defer ces.backendMutex.Unlock()
-	timeSinceCESQueued := time.Since(ces.cesInsertedAt)
-	if !ces.cesInsertedAt.IsZero() {
-		var t time.Time
-		// Reset the cesInsertedAt value
-		ces.cesInsertedAt = t
-		diff = timeSinceCESQueued.Seconds()
+	if ces, exists := c.desiredCESs.getCESTracker(cesName); exists {
```

Comment:
```suggestion
	ces, exists := c.desiredCESs.getCESTracker(cesName)
	if !exists {
	  return
	}
```

Reply:
I'd suggest to change quickly return if ces doesn't exist to improve code structure, to make it more flat.

Here there is one ces but let say we would have 3 objects. The current code would look:
```
if obj1, exists := getObj1(); exist {
  some code
  if obj2, exists := getObj2(); exist {
    some code
    if obj3, exists := getObj3(); exist {
      some code
}
return
```
instead of
```
obj1, exists := getObj1()
if !exists {
  return
}
some code
obj2, exists := getObj2()
if !exists {
  return
}
some code
obj3, exists := getObj3()
if !exists {
  return
}
some code
```

Reply:
I updated it as you suggested and made the same change in getCESMetricCountersAndClear() for consistency.